app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -131,16 +133,23 @@ end
 -%>
 
 rootpw --iscrypted <%= root_pass %>
+<%# Firewall -%>
 <% if host_param_true?('disable-firewall') -%>
 firewall --disable
 <% else -%>
 firewall --<%= os_major >= 6 ? 'service=' : '' %>ssh
 <% end -%>
+<%# Kdump -%>
+<% if host_param('kdump-options') -%>
```

Comment:
Only question I have is about empty strings. Should we ignore it when it's empty? If so I think this should work
```suggestion
<% if host_param('kdump-options').presence -%>
```

Reply:
Done, thank you